chatwoot/chatwoot
# feat: Sets up portal public views with rails ERB and tailwind

This PR sets up basic configurations to render help center home, articles.
This also includes 
1. rails turbo links
2. tailwind css

**How to test**
1. Navigate to portals home page and make sure you have an active portal. If not create one
2. Goto `http://localhost:3000/public/api/v1/portals/portal-slug`

<img width="3003" alt="image" src="https://user-images.githubusercontent.com/1277421/188165874-142eac74-b34a-4554-9687-08e51a4ecf00.png">
<img width="3003" alt="image" src="https://user-images.githubusercontent.com/1277421/188165968-a2042d11-1e9c-4787-b6e4-bc221bf0b2c8.png">
